test/e2e/cypress/support/commands.js

```diff
@@ -37,6 +37,29 @@ Cypress.Commands.add("AuthLogin", () => {
    });
 });
 
+Cypress.Commands.add("DCSetCursor", (idDC, idRow) => {
+   cy.window().then((win) => {
+      let AB = win.AB;
+      if (AB) {
+         return AB.datacollectionByID(idDC).setCursor(idRow);
+         // Question: Do you have any Idea how to make this wait for
+         // the DC to be initialized before continuing?
```

Comment:
I think just use [`cy.wrap()`](https://docs.cypress.io/api/commands/wrap#Promises) to wrap the promise. Something like:

```js
const DC = AB.datacollectionByID(idDC);
const settingCursor = new Promise((done) => {
   if (DC.dataStatus == DC.dataStatusFlat.initialized) {
      DC.setCursor(idRow);
      done();
   } else {
      DC.once("initializedData", ()=>{
         DC.setCursor(idRow);
         done();
      });
   }
});
return cy.wrap(settingCursor);
```